tests/extraction_tests/resources/mock_sheet_data/start_page_mock.csv

```diff
@@ -3,7 +3,7 @@
 ,,,,,,,,,
 ,,,,,,,,,
 ,Reporting Period: ,,,,,,,,
-0,1 October 2022 to 31 March 2023,,,,,,,,
+0,1 April 2023 to 30 September 2023,,,,,,,,
```

Comment:
does this mean the tests have effectively been changed from round 3 to round 4?

Reply:
We should probably leave the data as is (testing round 3) and extend as necessary, on import, for further subsequent rounds. That way keep all the historical testing intact.

Reply:
reverted, this in my latest push 👍